Editor/EditorCore/EditorMeshHandles.cs

```diff
@@ -238,35 +238,33 @@ public static void DrawSceneSelection(SceneSelection selection)
             var positions = mesh.positionsInternal;
 
             // Draw nearest edge
-            if (selection.face != null)
+            foreach (var face in selection.faces)
```

Comment:
Suggest moving the `foreach(var face in selection.faces)` loop into the `TriangleDrawingScope`. It saves you from setting up the triangle drawing scope and matrix for each face.